azure-pipelines.yml

```diff
@@ -11,8 +11,9 @@ jobs:
         # https://github.com/Homebrew/homebrew-bundle/pull/384
         # brew style homebrew/bundle
         # Use local config until then
-        rubocop --force-exclusion --parallel --except FormulaAudit,FormulaAuditStrict,NewFormulaAudit --format simple
+        gem install bundler:1.16.6
```

Comment:
Why not use the latest version?

Reply:
[Tried](https://github.com/Homebrew/homebrew-bundle/pull/429/commits/f19402e486f597ae045d73485985e5ad3c002d1f#diff-fec826feae04e51c0d94076385408bdcR14), didn't work. Easier to set version explicitly and revisit when I had a larger swath of time available. I was starting a job interview in less than two minutes from when I noticed the CI failures and even used the first minute of the job interview to explain what I was doing!